### DISCUSSION TITLE
GitHub Universe 2020 Day 1: What did you learn?

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#523] [Conference Notes]
From the [GitHub Universe 2020](http://githubuniverse.com) please share what you learnt or any tools you want to learn more about below👇 

<img width="1656" alt="Screenshot 2020-12-07 at 21 54 08" src="https://user-images.githubusercontent.com/624760/101410113-c5587e80-38d6-11eb-9d34-fe8e20f70a55.png">

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 1](https://www.twitch.tv/githubeducation)**  🎉

# **`The move to industry: your first job`** :office: 

## [Learning by teaching for your community](https://embed.twitch.tv/?height=800&migration=true&parent=education.github.com&referrer=https%3A%2F%2Feducation.github.com%2Funiversity%2F&video=830394431&width=100%25) :teacher: 
**Cassidy Williams - Principal Developer Experience Engineer, Netlify**
> When you're learning anything, but particularly technical content, the best way to get better at it is with others! In this session, Cassidy will talk about the importance of community, learning by teaching, and strategies for how to not be intimidated at the concept of combining the two: teaching your community to learn better.

Gist of the talk :studio_microphone:  

> Mentorship
 - Find a guiding mentor
 - Be open to your mentor
 - Share your thoughts with your mentor
 - Reach out to them suggestions/feedbacks

> Teaching works: Teaching can be intimidating, but it doesn’t have to be.
 - Writing Blogs
 - Making Short Videos
 - Answering on stack overflow


> Repetition is recognition

> Be a part of Community
 - Actively Teach your community
 - Have a giving nature
 - Help your community members
 - Try to make it more inclusive

> Know and observe your surroundings
> Learn from your seniors
> Not to be afraid to say you don’t know

If you have any questions, reach out to **[@cassidoo](https://twitter.com/cassidoo)**

> Do add your notes as well :smile:

**Reply by eddiejaoude** [reply]
This is EPIC thank you 👍

**Reply by siddharthdayalwal** [reply]
Thanks! :tada:

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 1](https://www.twitch.tv/githubeducation)**  🎉

# **`The move to industry: your first job`** :office: 

## [The myth of innate technical ability](https://embed.twitch.tv/?height=800&migration=true&parent=education.github.com&referrer=https%3A%2F%2Feducation.github.com%2Funiversity%2F&video=830394432&width=100%25) :technologist:  
**Omoju Miller - Technical Advisor to the CEO, GitHub**
> Omoju Miller deconstructs the myth of innate ability in science and tech, and explains the process of acquiring knowledge through logic and emotions that could allow anyone to join the AI revolution.

Gist of the talk :studio_microphone:  

> There is a importance of emotions in learning and AI.

> Humans are learning machines.

> Knowledge is acquired .
 - There are multiple ways of gaining knowledge.

> Getting into tech if coming from other background.
 - Learn to program.
 - Learn a programming language.
 - Start exploring.
 - Move forward with your interest.

> How to bring emotional connections on job learning?
 - Be curious.
 - Create the connection.
 - Put yourself in the place of other person.
 - Ask more questions.
 - Know your user.

> How to make people engage into learning/coding ?
 - Know the problem people have.
 - Know the motive of learning.
 - Build something that solves their problem.
 - Make them see the value.

> Internships Opportunities
 - MLH Fellowship
 - Google Summer of Code
 - Others (Explore yourself)

> Which area to focus in CS
 - Try different things
 - Keep a note on where you enjoyed

> Books or Resource to be a better Programmer
 - [A Mind for Numbers](https://www.amazon.in/Mind-Numbers-Science-Flunked-Algebra/dp/039916524X) 

If you have any questions, reach out to **[@omojumiller](https://twitter.com/omojumiller)**

> Do add your notes as well :smile:

**Reply by eddiejaoude** [reply]
You are on fire 💪

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 1](https://www.twitch.tv/githubeducation)**  🎉

# **`The move to industry: your first job`** :office: 

## [Everything you think you know about tech internships is wrong](https://embed.twitch.tv/?height=800&migration=true&parent=education.github.com&referrer=https%3A%2F%2Feducation.github.com%2Funiversity%2F&video=830404919&width=100%25) :nerd_face:   
**Mike Swift - CEO & Co-Founder, Major League Hacking (MLH)**
> Before COVID, landing a tech internship was always THE way to launch a career in software engineering. Today, things have changed. Massive shifts in remote work, online education, and the global economy have made things a lot more uncertain for both aspiring interns and employers. During this session, Swift will share insights from running the largest community of early career developers in the world, and lessons learned from teaming up with GitHub to reimagine the future of tech internships.

Gist of the talk :studio_microphone:  

> Learning to code is like getting from zero to one
> MLH helps developers get from one to a job

> 40,000+ Lost Internships due to pandemic

> What do tech-internships look like in post-COVID world?
 - Normal Path : School --> First Job
   - There is a big gap between learning and implementing.
   - School --> Gap-1 --> First Job
 - Education is hard, Technical Education is harder.
 - Coding is more art than science.
   - It needs practice and practice.

> How to bridge the Gap-1 ?? :thinking: 
   - Let students explore new things.
   - Let students interact with employers
   - Let them try and solve real life problems
   - Updated path : School -->  Gap-2 --> **Internship** --> First Job

> Projects + Managements + Mentor-ship + Experiences + Sourcing + Screening 
== Hard To Scale

> Resume filters don't work when there's nothing on your resume.

> How to bridge Gap-2? :thinking: 
 - School -->  **Hackathons** --> Internship --> First Job
 - Hackathons : 
A hackathon (also known as a hack day, hackfest or codefest; a portmanteau of hacking marathon) is a design sprint-like event; often, in which computer programmers and others involved in software development, including graphic designers, interface designers, project managers, domain experts, and others collaborate intensively on software projects.
   - Best place to turn your ideas into reality.
   - Collaborate with people.
   - Make connections.
   - Interact with companies and industry experts.

> Hackathons != Productions
 - School -->  Hackathons --> Gap-3 --> Internship --> First Job

> How to bridge Gap-3? :thinking: 
 - School -->  Hackathons --> **Open-Source** --> Internship --> First Job
  - 73% of hackers want to contribute to OSS but do not know how to start.

> Challenges 
 - In-Person
 - Time Intensive
 - Expensive
 - Difficult
 - Hard to scale

> How to bridge Gap-1,2,3 together and solve above challenges?
- **[MLH Fellowship - Powered by GitHub](https://fellowship.mlh.io/)**
- 30,000+ Developers Applied

> Happy Hacking | [Major League Hacking](https://mlh.io/)


If you have any questions, reach out to **[@SwiftAlphaOne](https://twitter.com/SwiftAlphaOne)**

> Do add your notes as well :smile:

**Reply by eddiejaoude** [reply]
WOW this is super awesome 👍

**Comment by siddharthdayalwal** [comment]
Will be sharing notes from **[GitHub Univers(ity) 2020 - Day 1](https://www.twitch.tv/githubeducation)**  🎉

# **`The move to industry: your first job`** :office: 

## [Boosting virtual learning environments using world-leading technologies](https://embed.twitch.tv/?height=800&migration=true&parent=education.github.com&referrer=https%3A%2F%2Feducation.github.com%2Funiversity%2F&video=830404918&width=100%25) :100:    
**John Faber Flórez Vasco - Principal Software Engineer, Agile Innova**
> Among the main challenges that the educational model must face is how to provide knowledge that is relevant, updated, and that corresponds to the reality that graduates will face in their professional performance. At the same time, they must build a proposal for the training process with didactic resources and pedagogical strategies that facilitate their learning. By integrating GitHub into the collaborative management of software development projects carried out by students, the advantages of using the tool were made appropriate by students and teachers, and it was the best option to overcome the restrictions that arose as a result of isolation derived from COVID-19. Taking advantage of the benefits of the GitHub API, the information from the repositories of 125 projects, developed by 670 high school seniors from 25 schools in Medellín, Colombia, was unified in web development.

> Talk was in Spanish, so I was not able to take notes!

If you have any questions, reach out to **[@faber_vasco](https://twitter.com/faber_vasco)**

> Do add your notes :smile:

**Reply by JohnFlorez25** [reply]
Thanks for the mention @siddharthdayalwal

**Reply by siddharthdayalwal** [reply]
Your welcome @JohnFlorez25 :+1:

**Comment by eddiejaoude** [comment]
You are always adding value 👍 💯

**Reply by siddharthdayalwal** [reply]
Happy to help always! :relieved:
